bundle/bundle.go

```diff
@@ -36,7 +37,14 @@ func New(path string, s *specs.Spec) (*Bundle, error) {
 }
 
 func Load(path string) (*Bundle, error) {
-	// TODO: do validation
+	fi, err := os.Stat(path)
```

Comment:
nit: can you use the `errors` package?

e.g. `errors.Wrap(err, "failed to stat")`

For the check below you can use `errors.Errorf()`